api/operator/v1beta1/vmextra_types.go

```diff
@@ -899,20 +899,14 @@ func (c *TLSConfig) BuildAssetPath(prefix, name, key string) string {
 	return fmt.Sprintf("%s_%s_%s", prefix, name, key)
 }
 
-// WebserverTLSConfig defines TLS configuration for the applications webserver
-type WebserverTLSConfig struct {
-	// ClientAuthType defines server policy for client authentication
-	// If you want to enable client authentication (aka mTLS), you need to use RequireAndVerifyClientCert
-	// Note, mTLS is supported only at enterprise version of VictoriaMetrics components
-	// +kubebuilder:validation:Enum=NoClientCert;RequireAndVerifyClientCert
-	ClientAuthType string `json:"client_auth_type,omitempty"`
-
-	// ClientCA defines reference for secret with CA content under given key
+// Certs defines TLS certs configuration
+type Certs struct {
+	// CA defines reference for secret with CA content under given key
 	// mutually exclusive with ClientCAFile
-	ClientCASecretRef *v1.SecretKeySelector `json:"client_ca_secret_ref,omitempty"`
-	// ClientCAFile defines path to the pre-mounted file with CA
-	// mutually exclusive with ClientCASecretRef
-	ClientCAFile string `json:"client_ca_file,omitempty"`
+	CASecretRef *v1.SecretKeySelector `json:"ca_secret_ref,omitempty"`
```

Comment:
I'd suggest to keep old naming, since it less confuses users. 

 For webserver tls config `client_ca_file` entry looks easy to understand and do not confuse, when you use documentation from http traffic configuration page https://prometheus.io/docs/alerting/latest/https/#http-traffic

 Yes, it requires some copy-paste. But it's acceptable trade-off.